src/stores/client_focus_store.ts

```diff
@@ -0,0 +1,71 @@
+import { Get } from "../store_engine";
+import { ClientId } from "../types";
+import { SpreadsheetStore } from "./spreadsheet_store";
+
+export class ClientFocusStore extends SpreadsheetStore {
+  mutators = [
+    "focusClient",
+    "unfocusClient",
+    "showClientTag",
+    "hideClientTag",
+    "jumpToClient",
+  ] as const;
+
+  private _showClientTag: Boolean = false;
```

Comment:
`Boolean` => `boolean`, use the typescript literal type instead of the Javascript Boolean constructor.
And in this case, TS infers the type for you